test/issue-636

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+AURDEST="$(mktemp -d --tmpdir "aurutils-${BASH_SOURCE[0]##*/}.XXXXXXXX")"
+export AURDEST
+
+cleanup() {
+    rm -rf --one-file-system --preserve-root "${AURDEST?}"
```

Comment:
A question since "cleanup" is used in other places with temporary files: why those extra flags to `rm`, if the target string is quoted?

Reply:
Two things: an abundance of caution and (marginally) better diagnostics (well, three, including force of habit).  Certainly not wedded to this approach, though, so I've harmonized this trap function with others in the existing codebase.